js/adapt-stateful-session.js

```diff
@@ -54,7 +54,7 @@ define([
       if (settings._suppressErrors) {
         this.scorm.suppressErrors = settings._suppressErrors;
       }
-      if (settings._commitOnStatusChange) {
+      if (settings._commitOnStatusChange === false) {
```

Comment:
```suggestion
      if (_.isBoolean(settings._commitOnStatusChange)) {
```
Might be a better solution - just on the off-chance we ever decide to change the default of `scorm.commitOnStatusChange` to `false`.

While you're there, would you mind making the same change for:
* `settings._suppressErrors`
* `settings._commitOnAnyChange`

?

Reply:
Yep, looks like a much better solution, thanks. Updated as requested.. :)